core/src/main/resources/lib/layout/icon.jelly

```diff
@@ -87,7 +88,7 @@ THE SOFTWARE.
         </j:when>
 
         <j:when test="${iconMetadata.svgSprite and !iconMetadata.buildStatus}">
-          <l:svgIcon class="${attrs.class}" href="${iconSrc}" id="${attrs.id}"/>
+          <l:svgIcon class="${attrs.class}" href="${iconSrc}" id="${attrs.id}" tooltip="${attrs.tooltip}"/>
```

Comment:
unrelated change?

Reply:
sort of. but all other places pass on the tooltip so why not there as well?

Reply:
not sure, i think it should be split out to another PR, at least the build status one works but it might have special handling.

Have you got a use-case for this?

Reply:
removed passing the tooltip as I don't need it